Dockerfile

```diff
@@ -52,7 +52,9 @@ COPY ./ /code/
 
 RUN python manage.py collectstatic --noinput
 
+ARG GIT_TAG=
 ARG GIT_COMMIT=
```

Comment:
Is there a reason for the separate GIT_TAG and GIT_COMMIT? It [looks like](https://github.com/CenterForOpenScience/SHARE/blob/develop/project/settings.py#L390) GIT_COMMIT is only used for `release` in sentry issues, and it looks like it's null. Seems like they could be the same thing?